README.md

```diff
@@ -390,7 +390,7 @@ task:
 heroku run -s standard-2x rails dss:publish_thesis_by_id[THESIS_ID] --app TARGET-HEROKU-APP
 ```
 
-Note: `Pending publication` is allowed here, but not expected to be a normal occurence, to handle the edge case of the app thinking data was sent to SQS but the data not arriving for any reason.
+Warning: `Pending publication` is allowed here, but not expected to be a normal occurence, to handle the edge case of the app thinking data was sent to SQS but the data not arriving for any reason. It is possible to send multiple copies of a single thesis to the publishing queue with this feature. Please be sure the SQS queues are fully processed and that you are absolutely confident that sending a `Pending publication` status thesis back into the publishing queue is appropriate. Failure to use care could lead to a single thesis being published more than once, which is bad. This is only a concern specifically for the `Pending publication` and `Publication review` statuses as no other statuses can be sent to the queue with this method.
```

Comment:
What are the implications of double-publishing? Multiple DSpace records?

Reply:
Ha. Great question and I agree it should be added here for context. I'm not actually even sure what would happen in our app in this situation, but it would definitely involve having the thesis team clean up the extra record. I'll leave this open until I can figure out how best to word the important context here.

Reply:
Fwiw, I think it would provide sufficient context to say something like, "which may result in duplicated theses being published and other unexpected consequences." (I don't think we necessarily need to document _everything_ that could go wrong.)